### PR TITLE
Port some more isolated docs

### PR DESCRIPTION
**Pull request by carlossanlop**
And fix some of the weird tags that have been showing up lately.

### CODE REVIEW

File: xml/System.Text/Ascii.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides static methods to assist in the utilization and manipulation of ASCII characters and buffers.</summary>
     <remarks>To be added.</remarks>
```
Comment (carlossanlop) [inline xml/System.Text/Ascii.xml:18]: @GrabYourPitchforks suggested adding this remark:

```suggestion
    <remarks>Within this class, an ASCII <see langword="byte"/> is defined as a <see cref="T:System.Byte"/> between <c>0x00</c> and <c>0x7F</c>, inclusive. An ASCII <see langword="char"/> is defined as a <see cref="T:System.Char"/> between <c>0x0000</c> and <c>0x007F</c>, inclusive.</remarks>

```

File: xml/System.Text.Unicode/Utf8.xml [hunk 2]
```diff
@@ -1,243 +1,246 @@
...
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.InterpolatedStringHandlerArgument("destination")&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="net-8.0" />
+      </Parameters>
+      <Docs>
+        <param name="destination">The span to which the interpolated string should be formatted.</param>
+        <param name="handler">The interpolated string.</param>
+        <param name="bytesWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System.Text.Unicode/Utf8.xml:192]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System.Text.Unicode/Utf8.xml [hunk 3]
```diff
@@ -1,243 +1,246 @@
...
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="net-8.0" />
+      </Parameters>
+      <Docs>
+        <param name="destination">The span to which the interpolated string should be formatted.</param>
+        <param name="provider">An object that supplies culture-specific formatting information.</param>
+        <param name="handler">The interpolated string.</param>
+        <param name="bytesWritten">The number of characters written to the span.</param>
```
Comment (gewarren) [inline xml/System.Text.Unicode/Utf8.xml:238]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the number of characters written to the span.</param>
```

File: xml/System.Text/ASCIIEncoding.xml [hunk 4]
```diff
@@ -1821,11 +1821,12 @@
...
       </Parameters>
       <Docs>
-        <param name="chars">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="chars">The span containing the set of characters to encode.</param>
+        <param name="bytes">The byte span to hold the encoded bytes.</param>
+        <param name="bytesWritten">Upon successful completion of the operation, the number of bytes encoded into <paramref name="bytes"/>.</param>
```
Comment (gewarren) [inline xml/System.Text/ASCIIEncoding.xml:1826]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the number of bytes encoded into <paramref name="bytes"/>.</param>
```

File: xml/System.Text/ASCIIEncoding.xml [hunk 5]
```diff
@@ -1854,11 +1855,12 @@
...
       </Parameters>
       <Docs>
-        <param name="bytes">To be added.</param>
-        <param name="chars">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="bytes">A read-only span containing the sequence of bytes to decode.</param>
+        <param name="chars">The character span receiving the decoded bytes.</param>
+        <param name="charsWritten">Upon successful completion of the operation, the number of chars decoded into <paramref name="chars"/>.</param>
```
Comment (gewarren) [inline xml/System.Text/ASCIIEncoding.xml:1860]:
```suggestion
        <param name="charsWritten">When this method returns, contains the number of chars decoded into <paramref name="chars"/>.</param>
```

File: xml/System.Threading.Tasks/ConfigureAwaitOptions.xml [hunk 6]
```diff
@@ -1,107 +1,107 @@
...
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Threading.Tasks.ConfigureAwaitOptions</ReturnType>
+      </ReturnValue>
+      <MemberValue>1</MemberValue>
+      <Docs>
+        <summary>Attempt to marshal the continuation back to the original <see cref="T:System.Threading.SynchronizationContext" /> or <see cref="T:System.Threading.Tasks.TaskScheduler" /> present on the originating thread at the time of the await.</summary>
```
Comment (gewarren) [inline xml/System.Threading.Tasks/ConfigureAwaitOptions.xml:43]:
```suggestion
        <summary>Attempts to marshal the continuation back to the original <see cref="T:System.Threading.SynchronizationContext" /> or <see cref="T:System.Threading.Tasks.TaskScheduler" /> present on the originating thread at the time of the await.</summary>
```

File: xml/System.Threading/PeriodicTimer.xml [hunk 7]
```diff
@@ -1,203 +1,209 @@
...
+          <AttributeName Language="C#">[System.Runtime.CompilerServices.NullableContext(1)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.NullableContext(1)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <Parameters>
+        <Parameter Name="period" Type="System.TimeSpan" Index="0" FrameworkAlternate="net-8.0" />
+        <Parameter Name="timeProvider" Type="System.TimeProvider" Index="1" FrameworkAlternate="net-8.0" />
+      </Parameters>
+      <Docs>
+        <param name="period">The period between ticks</param>
```
Comment (gewarren) [inline xml/System.Threading/PeriodicTimer.xml:85]:
```suggestion
        <param name="period">The period between ticks.</param>
```

File: xml/System.Xml/XmlResolver.xml [hunk 8]
```diff
@@ -249,9 +249,9 @@ resolver.Credentials = myCache;
         <ReturnType>System.Xml.XmlResolver</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets an XML resolver which resolves only file system URIs.</summary>
```
Comment (gewarren) [inline xml/System.Xml/XmlResolver.xml:252]:
```suggestion
        <summary>Gets an XML resolver that resolves only file system URIs.</summary
```

File: xml/System.Xml/XmlResolver.xml [hunk 9]
```diff
@@ -249,9 +249,9 @@ resolver.Credentials = myCache;
...
+        <value>An XML resolver which resolves only file system URIs.</value>
```
Comment (gewarren) [inline xml/System.Xml/XmlResolver.xml:253]:
```suggestion
        <value>An XML resolver that resolves only file system URIs.</value>
```

File: xml/System.Xml/XmlResolver.xml [hunk 10]
```diff
@@ -249,9 +249,9 @@ resolver.Credentials = myCache;
...
+        <remarks>Calling <see cref="M:System.Xml.XmlResolver.GetEntity(System.Uri,System.String,System.Type)"/> or <see cref="M:System.Xml.XmlResolver.GetEntityAsync(System.Uri,System.String,System.Type)"/> on the <see cref="T:System.Xml.XmlResolver"/> instance returned by this property will resolve only URIs which scheme is file.</remarks>
```
Comment (gewarren) [inline xml/System.Xml/XmlResolver.xml:254]:
```suggestion
        <remarks>Calling <see cref="M:System.Xml.XmlResolver.GetEntity(System.Uri,System.String,System.Type)"/> or <see cref="M:System.Xml.XmlResolver.GetEntityAsync(System.Uri,System.String,System.Type)"/> on the <see cref="T:System.Xml.XmlResolver"/> instance returned by this property will resolve only URIs whose scheme is file.</remarks>
```

**Review by gewarren — APPROVED** [review]

**Comment by carlossanlop** [comment]
Oof... the conflicts are complicated to solve. They are showing up after we updated the CI with RC1. I'll see what I can do.

**Comment by gewarren** [comment]
> Oof... the conflicts are complicated to solve. They are showing up after we updated the CI with RC1. I'll see what I can do.

Yes, it's because of the line ending differences I think.

**Comment by carlossanlop** [comment]
@gewarren I'll close this because it's already quite noisy. The tool re-run added many more files so I'll have to split the change in a few more PRs to make it easier to review.

**Pull request closed without merging**